### PR TITLE
Huff bio

### PR DESCRIPTION
**Pull request by katyhuff**
Finished my bio. Also, changed my name in the proposal document to my longer, fancier, more technical-publication-friendly name, Kathryn. I rebuilt my bio, but didn't rebuild the proposal (some font thing broke it for me?).

### CODE REVIEW

File: huff-kathryn-bio.tex [hunk 1]
```diff
@@ -187,6 +187,11 @@ \subsection{Products}
           White,~E.P., Wilson,~G.V., and Wilson,~P.P.H.  ``Best Practices For 
           Scientific Computing.'' \textsl{PLOS Biology}, Vol 1, Issue 12, 2014. 
           arXiv:1210.0530 [cs.MS].
```
Comment (gonuke) [inline huff-kathryn-bio.tex:189]: Is the DOI better than an arXiv reference here?

File: huff-kathryn-bio.tex [hunk 2]
```diff
@@ -187,6 +187,11 @@ \subsection{Products}
...
+
+   \item \textbf{Huff K.}, Scopatz, A., Preston, N., Wilson, P.P.H. ``Rapid
+          Peer Education of a Computational Nuclear Engineering Skill Suite.''
+          \emph{Transactions of the American Nuclear Society Annual Conference.}
+          Hollywood, FL. June 2011.
```
Comment (gonuke) [inline huff-kathryn-bio.tex:194]: Transactions publishes volumes and page numbers for a more archival looking citation.  This one is vol 104, pp. 103-104.

File: huff-kathryn-bio.tex [hunk 3]
```diff
@@ -195,23 +200,29 @@ \subsection{Synergistic Activities}
 
     \item \textbf{Conference service:}
```
Comment (gonuke) [inline huff-kathryn-bio.tex:201]: Recommend "Conference Program Committees": evokes more of the scientific peer notion vs. an organizational staffer

**Comment by gonuke** [comment]
I only noticed the references when I went to look there to poach some for my bio!! :)  Sorry to be late to this party.

File: huff-kathryn-bio.tex (hunk 3 above)
Comment (katyhuff) [inline huff-kathryn-bio.tex:201]: Thanks for these comments paul! Pushed 'em!

**Pull request merged**